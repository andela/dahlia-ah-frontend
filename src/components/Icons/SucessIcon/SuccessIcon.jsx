import React from 'react';
import './SuccessIcon.scss';

const SuccessIcon = () => (
  <>
    <img className="success-icon" alt="success icon" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAQAAADa613fAAAAAmJLR0QA/4ePzL8AAAdHSURBVHja7VxZbFRVGP46ZQaQrVTC0kKXYSktlKJgQHwwkUWUphWUiFBks02otIhLiyYaIDapFEkKRR4ENAKiKLQBjUCRBlowvCgiGAXKThDaaZStDMuMDz33nzNtZ+659557Z4zz35dm5j//OV/v+fdzBohQhCIUof83RSMN07AUm3AYx1GPJrjhRhPqcRx12ISlmIpURIczBCfysB0ueAWem6hGMUYhKrwgJKAYp4UAtH4uohSDwwFCB+TgMDy6QCiPB3WYhQ6hA9EReahvZ2HXsAflyMd4pMOJnnDAgZ5wIh0TkI812Ivr7Yw6g9fgsB5EFObgcqul3MJOFGC4wL6PQjoKUYXbrSRcQo61WjME1X4LeIg65KGbZjmdMR278cBP1kEMtwaEA6W4x018B2uRaEhiEirQzEm8hxLYzYaRhKN+m+kj9JEity/KcIuTfAQJZsLI8vMSuw2+idYUjy846X/jJbNgvMeZ2XpMNmWOKTjLmeViM6xUGfff2oEY0956N2zjZloDm0zhdnzJKXee6QZlIaf8W+Upvo2DcR2jLbGNY9DAQZH0VlaTyPNIscxXDcQZmnedHBVXxJ1AnKXRQzxO0tyG1T6TLNVFDLA8EIrDObJg2cbC80YmqMHCTcXTIFxjK2hCkv5g5ChZqtEIFY3FXfL2Ou1XKe3QPISS8mkdJXqGD6PQ8OuQp29b2UrcSNXuyQ9RMNIt5EB6UOByQGu+Mpde55SwqApMpvXM1JbEKtnft2FT4KgiN6AhIc6jfMM63/EcLsMVJHRPpMR4vnhlRCkplFoGIxk3WCwXmFaxVZ0SLe/lkPfoYxEMG2rYnFeDcPWjmFhQTw4z9rWWvY/FpMq5QfnWMa5asaDAwyojiRbBGEK7/wcV45qMhyzycqqLXc6EVlu2rWopS1c3LcoW/EDdESquZ45FQJbStpotwD2f8Z5Wc4wjyPBa48/TSIGrhPi70zZMC85YyNh2WgLDjp8phe4tOGYXG7FIzH8WWAJkOW2r6cJjllAlJwhFo4mxDbMAxmMUX2/RMCqDjXEFK0mkUWPA/Jp4J5xgs11BrKa4XKmvDA3MNI2x7LHgfazUHV8rnYAgWfy7jKXcdBhPUjPhU81jK9jIosAsnzOWfJNhPIJT1NqJ0Ty6gI3dGJhF8bHjTQayloo8emaaSC2hgPQbYxHvGXVBZ80LeYZqZfrCUsVp/xqYRSmHiVaQFqAZN9VcUxvffJ7CjC46c5eW8WcDsyjluEcFRSr8H2pYxgY25gGe0rkxe1HRMCC5GYtoVvwHmdBPBCvmz9OIlbo1rCOTcFcekHEsQfXCi20Co2JxhXGfRCczgSgdQnFPO4pr/u9HV8FC2308YcDmCWwtRQ215IZDcZGgHA2qXdnEt8xgoUJV2bWbXwBI4HQlcA+lF/5iPL8YPKohYH4VhzhBo+jelFd4cRrJ7fLspJ2dbtCdTlJ3iJ/pDlG6coc6rmJEm+9ny+s9UYiyQT1/LtdlS3bQYpswzu+7ODIjP0k4O6cEje8EZplqKIyPxibucMez3Dffs09vY4iESE1591nqidV1nYmVjcJBL+7iRfZpLn22WEr5SCCx8qW6+o8aFdOyHyAXQBI5zVop/fKRIqkuUMnYCg1MVUjRrQdFOMD+vhHAmmmlN0WKD75yUJWhyWbjfpujfrL6kLuZvNeDs6WTWhor0GXijh+MfZLKGd1JbqpajULpjcw1OOXT+Iczx/GS3scC6pGo0jLG+qPhSR+nZn+OtCT5IJP4vjrrQGorJBmeNgUX4EWlNBhOtjaPmOGoY6grJEwdhxqJfa/16lEWT7MYezP6SphcXs0yjg5zzBANNU4ZTkfNoNV0Zlv4+HkuRUwJYQMjmQzvPPFBDlxigyrDBojSFzmn7YyQL3/IDAsYWbSel7WqaA1lxj1CDiOGqgn7tQ9OozbM9pAD2UHHnIbqGV5Cr3NhSGEU0DpW6BNgxxHyKGNCBmMceY86/bd+BlB1t1H72TUplEbZvsvYSYwplCRdCoFPiScl9wTL0LWmrielBeNi1B+/09xvyxD4MYm7Yri4Jk6puEDzlskRaaPisxcNGGuRivuu3GyWF3jasYUr8+SbDqOALJUXm+Xet4ri+uJeVKKnaSC64ytupnK5F2EUtfddTTpnUgyWRXbKCw/eMuu/lUl+peWyWJJkK/UNJ73R3BPHCeTtW4pGq9BPUva32q+AVIf+ZiuiHSV+Fyqbsc5gDdGJ9Zxye+HGCquuIA/G3jZXXBcLN7V91AOvorrV3esatbNxsmkm57CUc8K78AYyBKy+DSOxBN/5XWxtMSEzEAKyYz6VKfinAftQgUWYiAw4EQsHHIiFExmYhEWowD7uRpvv+RPzzL+hG6ziMhO1hi/mH8Ir4fErEANQ3O67UX8uoBSDEGbU8uMVjUIAboTnj1f4K3IKXkARNqIWx1APF9xww4V6HEMtNqAI2UgxI/CIUIQiFKH/Ev0LvBsV7LAoUJYAAAAASUVORK5CYII=" />
  </>
);

export default SuccessIcon;
