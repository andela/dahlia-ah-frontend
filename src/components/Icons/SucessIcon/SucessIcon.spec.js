import React from 'react';
import { configure, mount } from 'enzyme';

import Adapter from 'enzyme-adapter-react-16';
import SucessIcon from './SuccessIcon';

configure({ adapter: new Adapter() });

describe('SucessIcon component <SucessIcon />', () => {
  it('renders correctly', () => {
    const wrapper = mount(
      <SucessIcon />,
    );

    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });

  it('has an image with the correct class and alt and src attributes ', () => {
    const wrapper = mount(
      <SucessIcon />,
    );

    expect(wrapper.find('.success-icon')).toHaveLength(1);
    expect(wrapper.find('.success-icon').get(0).props.alt).toEqual('success icon');
    expect(wrapper.find('.success-icon').get(0).props.src).toBeDefined();
    wrapper.unmount();
  });
});
