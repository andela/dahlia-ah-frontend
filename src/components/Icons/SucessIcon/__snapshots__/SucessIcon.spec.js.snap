// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SucessIcon component <SucessIcon /> renders correctly 1`] = `ReactWrapper {}`;
